infra/ansible/e2e_tests.Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN ansible-playbook -vvv playbook.yaml -e "stage=release" -e "${ansible_vars}"
 # Copy test sources.
 RUN mkdir -p /src/pytorch/xla
 COPY --from=build /src/pytorch/xla/test /src/pytorch/xla/test
+# Copy ci_commit_pins from upstream
+RUN mkdir -p /src/pytorch/.github
+COPY --from=build /src/pytorch/.github/ci_commit_pins /src/pytorch/xla/test/ci_commit_pins
```

Comment:
nit: can you run the torchvision install in the Dockerfile?

Reply:
Tried to install in Dockerfile, but looks like installing torchvision in this way requires torch to be already installed. So we have to install after torch is compiled.
```
Step #2 - "build_pytorch-xla-test_docker_image": Step 19/26 : RUN pip install --no-use-pep517 "git+https://github.com/pytorch/vision.git@$(cat /src/pytorch/.github/ci_commit_pins/vision.txt)"
Step #2 - "build_pytorch-xla-test_docker_image":  ---> Running in 5513d8dbc80a
Step #2 - "build_pytorch-xla-test_docker_image": Collecting git+https://github.com/pytorch/vision.git@2c127da8b5e2e8f44b50994c6cb931bcca267cfe
Step #2 - "build_pytorch-xla-test_docker_image":   Cloning https://github.com/pytorch/vision.git (to revision 2c127da8b5e2e8f44b50994c6cb931bcca267cfe) to /tmp/pip-req-build-th1jpkpj
Step #2 - "build_pytorch-xla-test_docker_image": �[91m  Running command git clone --filter=blob:none --quiet https://github.com/pytorch/vision.git /tmp/pip-req-build-th1jpkpj
Step #2 - "build_pytorch-xla-test_docker_image": �[0m�[91m  Running command git rev-parse -q --verify 'sha^2c127da8b5e2e8f44b50994c6cb931bcca267cfe'
Step #2 - "build_pytorch-xla-test_docker_image": �[0m�[91m  Running command git fetch -q https://github.com/pytorch/vision.git 2c127da8b5e2e8f44b50994c6cb931bcca267cfe
Step #2 - "build_pytorch-xla-test_docker_image": �[0m�[91m  Running command git checkout -q 2c127da8b5e2e8f44b50994c6cb931bcca267cfe
Step #2 - "build_pytorch-xla-test_docker_image": �[0m  Resolved https://github.com/pytorch/vision.git to commit 2c127da8b5e2e8f44b50994c6cb931bcca267cfe
Step #2 - "build_pytorch-xla-test_docker_image":   Preparing metadata (setup.py): started
Step #2 - "build_pytorch-xla-test_docker_image":   Preparing metadata (setup.py): finished with status 'error'
Step #2 - "build_pytorch-xla-test_docker_image": �[91m  error: subprocess-exited-with-error
Step #2 - "build_pytorch-xla-test_docker_image":   
Step #2 - "build_pytorch-xla-test_docker_image":   × python setup.py egg_info did not run successfully.
Step #2 - "build_pytorch-xla-test_docker_image":   │ exit code: 1
Step #2 - "build_pytorch-xla-test_docker_image":   ╰─> [6 lines of output]
Step #2 - "build_pytorch-xla-test_docker_image":       Traceback (most recent call last):
Step #2 - "build_pytorch-xla-test_docker_image":         File "<string>", line 2, in <module>
Step #2 - "build_pytorch-xla-test_docker_image":         File "<pip-setuptools-caller>", line 34, in <module>
Step #2 - "build_pytorch-xla-test_docker_image":         File "/tmp/pip-req-build-th1jpkpj/setup.py", line 9, in <module>
Step #2 - "build_pytorch-xla-test_docker_image":           import torch
Step #2 - "build_pytorch-xla-test_docker_image":       ModuleNotFoundError: No module named 'torch'
Step #2 - "build_pytorch-xla-test_docker_image":       [end of output]
Step #2 - "build_pytorch-xla-test_docker_image":   
Step #2 - "build_pytorch-xla-test_docker_image":   note: This error originates from a subprocess, and is likely not a problem with pip.
Step #2 - "build_pytorch-xla-test_docker_image": �[0m�[91merror: metadata-generation-failed
Step #2 - "build_pytorch-xla-test_docker_image": 
Step #2 - "build_pytorch-xla-test_docker_image": × Encountered error while generating package metadata.
Step #2 - "build_pytorch-xla-test_docker_image": ╰─> See above for output.
Step #2 - "build_pytorch-xla-test_docker_image": 
Step #2 - "build_pytorch-xla-test_docker_image": note: This is an issue with the package mentioned above, not pip.
Step #2 - "build_pytorch-xla-test_docker_image": hint: See above for details.
Step #2 - "build_pytorch-xla-test_docker_image": �[0m�[91m
Step #2 - "build_pytorch-xla-test_docker_image": [notice] A new release of pip is available: 23.0.1 -> 24.0
Step #2 - "build_pytorch-xla-test_docker_image": [notice] To update, run: pip install --upgrade pip
Step #2 - "build_pytorch-xla-test_docker_image": The command '/bin/sh -c pip install --no-use-pep517 "git+https://github.com/pytorch/vision.git@$(cat /src/pytorch/.github/ci_commit_pins/vision.txt)"' returned a non-zero code: 1
```

Reply:
I guess that's fine. The GCB-based CI is on its way out anyway